codegenerator/cli/templates/static/codegen/src/EventProcessing.res

```diff
@@ -535,6 +535,7 @@ let getDynamicContractRegistrations = (
       ~inMemoryStore,
       ~isInReorgThreshold=false,
       ~config,
+      ~safeChainIdAndBlockNumberArray=[], //No need to prune history for dynamic contract pre registration
```

Comment:
The naming is confusing. Maybe I'll understand what it means after reviewing further, but for now, I can only guess.